website/blog/2020-03-21-2.0.0.md

```diff
@@ -1724,11 +1724,11 @@ type Type2 implements A, B & C {
    d
     e
 
-<!-- Prettier stable -->
-- a
-   b
-   c
-   d
+<!-- Prettier 1.19 -->
+- a  
```

Comment:
@thorn0  I'm fixing https://github.com/prettier/prettier/commit/4314e9cacc42f545ef793a0fa105a91343af5c40#r37965119 here if that helps